### PR TITLE
Content types for stream and byoip

### PR DESCRIPTION
**Pull request by kodster28**
Leaving out SSL b/c it's such a beast...

Added `pcx-content-type` metadata to two doc products. Also fixed broken links + separated out the get started sections from the main page.

### CODE REVIEW

**Review by bkrebsbach — APPROVED** [review]

File: products/stream/src/content/get-started/index.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+---
+order: 1
+title: Get started
+pcx-content-type: how to
```
Comment (Everlag) [inline products/stream/src/content/get-started/index.md]: What does `pcx-content-type` do? ie, does it change the order on the sidebar, effect search, etc?

Its not clear to me what impact this change has to folks looking at the docs.

File: products/stream/src/content/index.md [hunk 2]
```diff
@@ -1,143 +1,17 @@
...
 <ButtonGroup>
-  <Button type="primary" href="https://dash.cloudflare.com/?to=/:account/stream">Visit the Stream dashboard</Button>
+  <Button type='primary' href='/get-started'>Get started</Button>
+  <Button type="secondary" href="https://dash.cloudflare.com/?to=/:account/stream">Visit the Stream dashboard</Button>
   <Button type="secondary" href="https://api.cloudflare.com/#stream-videos-properties">View API reference</Button>
 </ButtonGroup>
-
-----------
-
-# Getting started
```
Comment (Everlag) [inline products/stream/src/content/index.md]: @zaidf Do we want to split the index page off from the actual useful part of it?

My perspective on this is that we'd be forcing users to make one more unnecessary click to go from a barebones product description to actually useful documentation.

**Review by Everlag — CHANGES_REQUESTED** [review]

**Comment by Everlag** [comment]
Please don't merge the Stream changes until @zaidf takes a look and approves. This is a big change to what people see at the first page of our product docs.

File: products/stream/src/content/uploading-videos/index.md [hunk 3]
```diff
@@ -1,7 +1,9 @@
 ---
+title: Upload videos
 order: 1
+pcx-content-type: navigation
 ---
 
-# Uploading videos
+# Upload videos to Cloudflare Stream
```
Comment (Everlag) [inline products/stream/src/content/uploading-videos/index.md]: This change isn't in the scope of the commit message.

File: products/stream/src/content/get-started/index.md (hunk 1 above)
Comment (kodster28) [reply products/stream/src/content/get-started/index.md]: It doesn't impact any visible part of the docs. The goal of including it is to put our metadata close to individual pages so it's less likely to get out of date.

File: products/stream/src/content/index.md (hunk 2 above)
Comment (kodster28) [reply products/stream/src/content/index.md]: Howdy @Everlag (and @zaidf),

We trying to bring the Stream docs — together with other product docs — in line with the customer journey (and our content strategy).

What that means is structuring the first page of each docs section to be a true [landing page](https://wiki.cfops.it/display/Prod/Landing+Page). These pages — best example in [cf-one](https://developers.cloudflare.com/cloudflare-one/) — help customers decide whether or not they want to use the product in the first place (and then get into the more technical details).

I'll admit that all of that content isn't currently there and it's indeed a `barebones product description` at this point, so open to either holding off on this specific change or working together to provide a more robust landing page.

File: products/stream/src/content/uploading-videos/index.md (hunk 3 above)
Comment (kodster28) [reply products/stream/src/content/uploading-videos/index.md]: Can remove this if you want. Was trying to add a bit more context to a few headings to help SEO.

**Comment by kodster28** [comment]
Howdy @Everlag, since it's taken @zaidf a while to respond and our team is trying to get these `pcx-content-type` values implemented across developer docs (doesn't affect any visible part of the docs), I went ahead and reverted the restructuring around the Stream homepage.

We can go back and discuss/address that in another PR.

**Review by Everlag — APPROVED** [review]

**Comment by Everlag** [comment]
LGTM, thanks for making the changes. Happy to continue discussing the other changes in a separate PR.

File: products/byoip/src/content/dynamic-advertisement/best-practices.md [hunk 4]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Best practices
 order: 2
+pcx-content-type: best practices
```
Comment (crwaters16) [inline products/byoip/src/content/dynamic-advertisement/best-practices.md]: hyphenate

File: products/byoip/src/content/dynamic-advertisement/configure-dynamic-advertisement.md [hunk 5]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Configure
 order: 3
+pcx-content-type: how to
```
Comment (crwaters16) [inline products/byoip/src/content/dynamic-advertisement/configure-dynamic-advertisement.md]: hyphenate

File: products/byoip/src/content/get-started/index.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
+---
+order: 1
+pcx-content-type: how to
```
Comment (crwaters16) [inline products/byoip/src/content/get-started/index.md]: hyphenate

File: products/byoip/src/content/get-started/index.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+
+At a high level, Cloudflare requires a service-specific configuration for your prefixes, as well as some requirements that are common to all BYOIP customers, regardless of service type. This documentation covers the requirements common to all products that are compatible with BYOIP (i.e. Magic Transit, CDN, and Spectrum).
+
+There are two major prerequisites before Cloudflare can begin the work of onboarding your IP space.
+
+First, all customers must ensure their [Internet Routing Registry (IRR)](/irr-records) records are up to date with the correct prefix/ASN information.
+
+Secondly, Cloudflare must receive a [Letter of Authorization](/loa) to announce your prefixes, which we will share with our transit partners as evidence that we are allowed to announce the route.
+
+Optionally, if you use the RPKI protocol to sign your routes, Cloudflare can support you there as well. Just let your account team know if you’re interested in using RPKI!
```
Comment (crwaters16) [inline products/byoip/src/content/get-started/index.md]: exclamation point?! ;)

File: products/byoip/src/content/get-started/index.md [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+
+Please see the subsequent sections to learn more.
```
Comment (crwaters16) [inline products/byoip/src/content/get-started/index.md]: strike "please"

File: products/byoip/src/content/get-started/index.md [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+
+Once onboarded, BGP announcements for customer prefixes can be controlled with the [Dynamic Advertisement API](/dynamic-advertisement). This API has other useful features and can also be used via the Cloudflare dashboard.
```
Comment (crwaters16) [inline products/byoip/src/content/get-started/index.md]: have we defined BGP anywhere/do we need to?

File: products/byoip/src/content/index.md [hunk 10]
```diff
@@ -1,6 +1,7 @@
 ---
-title: Welcome
+title: Overview
 order: 0
+pcx-content-type: landing page
```
Comment (crwaters16) [inline products/byoip/src/content/index.md]: hyphenate

File: products/stream/src/content/uploading-videos/adding-captions.md [hunk 11]
```diff
@@ -1,5 +1,6 @@
 ---
 order: 4
+pcx-content-type: how to
```
Comment (crwaters16) [inline products/stream/src/content/uploading-videos/adding-captions.md]: this definitely should be a how-to, but its current state is messy (though we don't have a how-to strategy to align to yet). Consider interim, or we can just make Márcio aware that some of these pages will need to get some TLC in the future.

File: products/stream/src/content/uploading-videos/direct-creator-uploads.md [hunk 12]
```diff
@@ -1,5 +1,6 @@
 ---
 order: 3
+pcx-content-type: how to
```
Comment (crwaters16) [inline products/stream/src/content/uploading-videos/direct-creator-uploads.md]: same thought here - definitely should be a how-to, but not structured into steps at the moment

File: products/stream/src/content/uploading-videos/upload-video-file.md [hunk 13]
```diff
@@ -69,7 +70,7 @@ Setting arbitrary metadata values in the `Upload-Metadata` header sets values th
 
 - `allowedorigins`
 
-  - A comma separated strings containing the domains. This will set the [allowed origins setting](/viewing-your-videos/securing-your-stream) for the video.
+  - A comma separated strings containing the domains. This will set the [allowed origins setting](../../viewing-videos/securing-your-stream#security-considerations) for the video.
```
Comment (crwaters16) [inline products/stream/src/content/uploading-videos/upload-video-file.md]: odd article/noun agreement - A comma separated string OR comma separated strings

**Review by crwaters16 — APPROVED** [review]
Be sure to comb through and hyphenate the two-word pcx-content-types. A few other nit comments included as well.

File: products/stream/src/content/index.md (hunk 2 above)
Comment (zaidf) [reply products/stream/src/content/index.md]: @kodster28 as Kyle said, we'd strongly prefer having useful, actionable info on the first page. I think the vision to make the first page be more of a landing page makes sense. Given this is an API product, a lot of the best api products have API howtos on the landing page (ie. see mux.com) so I think a lot of what we have is already congruent with that vision. We can add more content on the page that a typical landing page might have while keeping the later sections.

**Pull request merged**